<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>530</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#backButton
{background-color: transparent;
border-image: url(:/menu/Images/Back.png);
background: none;
border: none;
background-repeat: none;}

#backButton:hover:!pressed
{
   border-image: url(:/menu/Images/hoverBack.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>350</width>
     <height>150</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="instructions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Our bodies are constantly working to protect us from various infectious agents like viruses, bacteria, and others. Some of the this immune battle can be felt, (like fevers), but much of it happens without our awareness. &lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Now, you'll take a direct part in helping cells defend against sickness.&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;As you defeat sicknesses, you will gain bone marrow. Bone marrow is where white blood cells are formed from stem cells, so as you will be able to use it to purchase white blood cell towers to defend yourself. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;You will be able to see the amount you have indicated on the right.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;To build a new white blood cell, simply click on the icon on the right side of the screen, then click on the map where you want it to be placed. Its cost in bone marrow will be subtracted from the amount you have.&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;You will have a set amount of &amp;quot;lives&amp;quot; called vitality. If you run out of vitality, you've gotten&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;sick and you lose the game.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;On the left is a progress bar. This will fill with defeated sicknesses as you progress through&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;each wave. If you defeat 10 waves, you win!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="howItWorksLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>541</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Our bodies are threatened by diseases caused by pathogens, like bacteria, viruses, and others. To protect us, the immune system works to fight pathogens. The immune system is divided in two parts: the innate immune system and the adaptive immune system.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Innate immunity is immunity that we are born with, and it includes the physical barriers which are the first line of defense, like skin and mucous. It also includes some non-specialized cells which can &amp;quot;eat&amp;quot; pathogens.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Adaptive immunity involves specialized immune cells which remember previously encountered pathogens to more efficiently attack and destroy them and the cells that they have infected. The adaptive immune system is what makes vaccines possible. Vaccines allow our adaptive immune system to get a head start on learning about a pathogen, which can be a life-saving time-savings in the race against multiplying pathogens.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#exitButton
{background-color: transparent;
border-image: url(:/menu/Images/Exit.png);
background: none;
border: none;
background-repeat: none;}

#exitButton:hover:!pressed
{
   border-image: url(:/menu/Images/hoverExit.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>150</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>241</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#playButton
{background-color: transparent;
border-image: url(:/menu/Images/Play!.png);
background: none;
border: none;
background-repeat: none;}

#playButton:hover:!pressed
{
   border-image: url(:/menu/Images/hoverPlay!.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>150</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="instructionsButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#instructionsButton
{background-color: transparent;
border-image: url(:/menu/Images/Instructions.png);
background: none;
border: none;
background-repeat: none;}

#instructionsButton:hover:!pressed
{
   border-image: url(:/menu/Images/hoverInstructions.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>800</width>
     <height>100</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="howButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>781</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#howButton
{background-color: transparent;
border-image: url(:/menu/Images/How Our Body Works.png);
background: none;
border: none;
background-repeat: none;}

#howButton:hover:!pressed
{
   border-image: url(:/menu/Images/hoverHow Our Body Works.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>900</width>
     <height>150</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>-110</x>
     <y>0</y>
     <width>821</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/menu/Images/Virus Tower Defense.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>651</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/menu/Images/menuBG.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="naturalKillerPicture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Natural Killer Cell</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/textures/Images/TD_NaturalKillerTower.png</normaloff>
     <disabledoff>:/textures/Images/TD_NaturalKillerTower.png</disabledoff>:/textures/Images/TD_NaturalKillerTower.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="memoryBPicture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Memory B Cell</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/textures/Images/TD_MemoryBTower.png</normaloff>
     <disabledoff>:/textures/Images/TD_MemoryBTower.png</disabledoff>:/textures/Images/TD_MemoryBTower.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="boneMarrowPicture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Bone Marrow</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/textures/Images/boneMarrow.png</normaloff>
     <disabledoff>:/textures/Images/boneMarrow.png</disabledoff>:/textures/Images/boneMarrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>instructions</zorder>
  <zorder>backButton</zorder>
  <zorder>howItWorksLabel</zorder>
  <zorder>exitButton</zorder>
  <zorder>playButton</zorder>
  <zorder>instructionsButton</zorder>
  <zorder>howButton</zorder>
  <zorder>titleLabel</zorder>
  <zorder>naturalKillerPicture</zorder>
  <zorder>memoryBPicture</zorder>
  <zorder>boneMarrowPicture</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
